core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/RawlsInstrumented.scala

```diff
@@ -55,6 +55,14 @@ trait RawlsInstrumented extends WorkbenchInstrumented {
       .expand(WorkflowStatusMetricKey, status)
       .transient()
       .asCounter("count")
+
+  /**
+    * A timer for capturing latency between initial Rawls submission and workflow processing in Cromwell.
+    */
+  protected def workflowToCromwellLatency: Timer =
+    ExpandedMetricBuilder
+      .expand(WorkspaceMetricKey, "submission_to_cromwell")
+      .asTimer("latency")
```

Comment:
A timer allows us to show mean elapsed time, 95th percentile, and standard deviation. For example,

![image](https://user-images.githubusercontent.com/484484/150007718-df11cfb8-8968-4548-a7b3-3a9bb38f727d.png)